src/main/java/org/aksw/gerbil/evaluate/impl/FMeasureCalculator.java

```diff
@@ -225,7 +209,15 @@ private double[] calculateMeasures(EvaluationCounts counts) {
         return new double[] { precision, recall, F1_score };
     }
 
-    private EvaluationResult[] getContingencyMatrix(EvaluationCounts counts[]){
-        return new EvaluationResult[] { new ObjectEvaluationResult(CONTINGENCY_MATRIX_NAME, counts)};
+    private AggregateContingencyMetricsReport updateContingencyMetricsReport(ExtendedContingencyMetrics[] extendedMetrics) {
```

Comment:
Two points here:
1. A class name should (in most cases) represent a thing while method names should tell what is done. So method names can start with a verb but class names typically shouldn't. Hence, "Aggregated..." would be the better start of the class name.
2. The method name does not really reflect what is done. The method does not "update" something, it creates an aggregation. It would be better to rename it accordingly

Reply:
Refactored Class and Method Names